<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestion</class>
 <widget class="QDialog" name="gestion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1281</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>1400</width>
     <height>1761</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Perpetua</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Gestion des citoyens:</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Cin</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>65</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom citoyen :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Prenom citoyen</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Email :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_cin">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nom">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_prenom">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>110</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_email">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>150</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ajoutC">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_pdf">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>380</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Imprimer</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="cinrecherche">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>340</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_actualiser_citoyen">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>380</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Actualiser</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_supprimer">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>300</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_modifier_citoyen">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>380</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>-10</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Perpetua</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Historique:</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>30</y>
       <width>301</width>
       <height>311</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_historiqueC">
     <property name="geometry">
      <rect>
       <x>994</x>
       <y>360</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Historique</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>430</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Contacter citoyen :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>480</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>texte:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_evoyer">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>450</y>
       <width>281</width>
       <height>91</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_mailingC">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>560</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Envoyer</string>
     </property>
    </widget>
    <widget class="QTableView" name="tabcitoyen">
     <property name="geometry">
      <rect>
       <x>275</x>
       <y>20</y>
       <width>521</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_tri_cin">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>360</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>tri par CIN </string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_trinom">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>390</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>tri par nom</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>350</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>recherche:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>-51</y>
       <width>1291</width>
       <height>761</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_13</zorder>
    <zorder>label</zorder>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>label_4</zorder>
    <zorder>cinrecherche</zorder>
    <zorder>label_12</zorder>
    <zorder>label_9</zorder>
    <zorder>label_15</zorder>
    <zorder>label_16</zorder>
    <zorder>lineEdit_cin</zorder>
    <zorder>lineEdit_email</zorder>
    <zorder>lineEdit_evoyer</zorder>
    <zorder>lineEdit_prenom</zorder>
    <zorder>lineEdit_nom</zorder>
    <zorder>pushButton_actualiser_citoyen</zorder>
    <zorder>pushButton_ajoutC</zorder>
    <zorder>pushButton_historiqueC</zorder>
    <zorder>pushButton_mailingC</zorder>
    <zorder>pushButton_modifier_citoyen</zorder>
    <zorder>pushButton_pdf</zorder>
    <zorder>radioButton_tri_cin</zorder>
    <zorder>radioButton_trinom</zorder>
    <zorder>tabcitoyen</zorder>
    <zorder>textBrowser</zorder>
    <zorder>pb_supprimer</zorder>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>Gestion des demandes :</string>
    </attribute>
    <widget class="QLineEdit" name="numD">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="typeD">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>160</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Num demande :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Id citoyen:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Date demande: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>160</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Type:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ajoutavis">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>210</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_modifieravis">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>290</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_suppavis">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>250</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_rechercheD">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>330</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_pdfavis">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>330</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Imprimer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_actualiser">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>330</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Actualiser</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_2">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>40</y>
       <width>331</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>10</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Historique:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_historiqueavis">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>330</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Historique</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>-4</x>
       <y>-47</y>
       <width>1111</width>
       <height>711</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="idcitoyen">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableView" name="tabledemande">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>41</y>
       <width>451</width>
       <height>261</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_triparnum">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>310</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>tri par num </string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_tridesc">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>340</y>
       <width>121</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>tri descendant</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>330</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>recherche:</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateD">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>120</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>380</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Description :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>380</y>
       <width>741</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
    <zorder>label_14</zorder>
    <zorder>numD</zorder>
    <zorder>typeD</zorder>
    <zorder>label_6</zorder>
    <zorder>label_7</zorder>
    <zorder>label_8</zorder>
    <zorder>label_10</zorder>
    <zorder>pushButton_ajoutavis</zorder>
    <zorder>pushButton_modifieravis</zorder>
    <zorder>pushButton_pdfavis</zorder>
    <zorder>textBrowser_2</zorder>
    <zorder>label_11</zorder>
    <zorder>pushButton_historiqueavis</zorder>
    <zorder>idcitoyen</zorder>
    <zorder>tabledemande</zorder>
    <zorder>radioButton_triparnum</zorder>
    <zorder>radioButton_tridesc</zorder>
    <zorder>label_17</zorder>
    <zorder>dateD</zorder>
    <zorder>label_5</zorder>
    <zorder>lineEdit</zorder>
    <zorder>pushButton_actualiser</zorder>
    <zorder>pushButton_suppavis</zorder>
    <zorder>lineEdit_rechercheD</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
